apps/verify/models/write-cookie.js

```diff
@@ -0,0 +1,47 @@
+'use strict';
+
+const dataService = require('../../common/data-service');
+const {
+  makeRequest,
+  requestHeaders,
+  } = dataService;
+
+/**
+ * API post - create cookie in the database
+ *
+ * @param {object} data - {created, uuid, userEmail} sent to the db
+ *
+ * @returns {Promise} response
+ */
+const sendDataToBeStored = async(data) => {
+  // current date
+  const created = (new Date()).toISOString();
+  const uuid = data.uuid;
+  const userEmail = data['user-email'];
+
+  const body = {
+    'created': created,
+    'uuid': uuid,
```

Comment:
```suggestion
    created,
    uuid,
```

Reply:
same will copy and paste so it's in one commit